department-of-veterans-affairs/caseflow
# Correspondence and Appeal Tasks: Related Appeals: Can view tasks added from Correspondence to existing appeals

Resolves [Correspondence and Appeal Tasks: Related Appeals: Can view tasks added from Correspondence to existing appeals](https://jira.devops.va.gov/browse/APPEALS-42529)

# Description
This PR introduces tasks added to appeals in the correspondence details section. Any Tasks added to appeals during an intake will populate in the matching section, giving a timeline as well as a table that will hold it about each of the tasks added.

## Acceptance Criteria

## Testing Plan
1. Navigate to the correspondence queue with user jon snow admin.
2. Take a correspondence through intake. On the tasks related to an appeal, add 1 or more.
3. For each of the appeals added, add at least more than one or more tasks to it and note the types and the description. Note that the default task that is added will not be populating in, and adding additional tasks with the "+add task" button is required.
4. Save the correspondence UUID at the top of the page.
5. Complete the intake and navigate to /queue/correspondence/UUID. Note that CAVC related tasks may cause an issue and fail to process, this is a bug unrelated to this work.
6. Note that the tasks added to appeal appear in the section below existing appeals.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1288" alt="image" src="https://github.com/user-attachments/assets/1d0a750b-48e4-4f3f-8760-7c6a8d707c6a">
 |<img width="647" alt="image" src="https://github.com/user-attachments/assets/d5960113-1cd4-4e3a-9ef5-6e4163e3ccc2">

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
